### PR TITLE
Always update wouldplays to avoid "same" command inconsistencies

### PR DESCRIPTION
**Pull request by yeanh16**

### CODE REVIEW

**Comment by yeanh16** [comment]
closes #46

File: handlers.py [hunk 1]
```diff
@@ -155,27 +151,26 @@ def split_string_by_keywords(self, string):
 
 
 class WouldPlayHandler(GameExtractionMixin, ContentBasedHandler):
-    fragments = ["I'd play", "id play", "I'd paly", "id paly", "I’d play", "I’d paly", "I’dplay", "I’dpaly", "same to", "same"]
```
Comment (JordanPowell) [inline handlers.py]: Why remove these?

Comment (yeanh16) [reply handlers.py]: "I’d play" and "I’d paly" are duplicates

**Comment by JordanPowell** [comment]
They have different apostrophe characters (specifically ones used on mobile
keyboards). I should have put that in the commit that added them. Oh well.

**Comment by yeanh16** [comment]
Added them back

**Pull request merged**